src/wallet/walletdb.cpp

```diff
@@ -219,7 +219,7 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
             CWalletTx wtx(nullptr /* pwallet */, MakeTransactionRef());
             ssValue >> wtx;
             CValidationState state;
```

Comment:
please remove the consensus #includes as well

Reply:
@achow101 ^

Reply:
```diff
 #include <wallet/walletdb.h>

-#include <consensus/tx_check.h>
-#include <consensus/validation.h>
 #include <fs.h>
 #include <key_io.h>
 #include <protocol.h>
```

Reply:
Done